vigetlabs/wordpress-site-starter
# [#9] Video Embed UI

# Summary

This PR adds UI cleanup to the **Video Embed** block.

## Issues

- #9 

## Testing Instructions

- Add a **Video Embed** block to your page
- Embed a video, and optionally, add a title and description paragraph
- If adding a heading and paragraph, verify that the spacing between them is `.5rem`, and spacing before the video is `3rem`

## Screenshots

![Screenshot 2024-04-26 at 1 31 23 PM](https://github.com/vigetlabs/wordpress-site-starter/assets/2685062/8d646c63-5d18-42d9-bc7c-e6c167dd62aa)
